lessons/ru/misc/nerves.md

```diff
@@ -0,0 +1,300 @@
+
+Обратите внимание, что здесь вам необходимо использовать параметры вашей локальной сети — в моей сети есть невыделенный IP-адрес `192.168.88.2`, который я собираюсь использовать.
+Однако в вашей сети невыделенный IP-адрес может отличаться.
+
+После внесения изменений нам потребуется записать измененную версию прошивки с помощью `mix firmware.burn`, а затем запустить устройство с картой, на которой записана новая версия прошивки.
+
+При включении устройства вы можете использовать команду `ping`, чтобы увидеть, что оно подключено к сети.
+
+```
+Request timeout for icmp_seq 206
+Request timeout for icmp_seq 207
+64 bytes from 192.168.88.2: icmp_seq=208 ttl=64 time=2.247 ms
+64 bytes from 192.168.88.2: icmp_seq=209 ttl=64 time=2.658 ms
+```
+
+Этот вывод означает, что устройство теперь доступно из сети.
+
+## Прошивка по сети
+
+До сих пор мы записывали SD-карты и физически загружали их в наше оборудование.
+Первое время этого достаточно, но гораздо проще устанавливать наши обновления по сети.
+Пакет [`ssh_subsystem_fwup`](https://github.com/nerves-project/ssh_subsystem_fwup) делает именно это.
+Он уже присутствует в вашем проекте по умолчанию и настроен на автоматическое обнаружение и поиск ключей SSH в вашем каталоге `~/.ssh`.
+
+Чтобы использовать функцию обновления прошивки по сети, вам необходимо сгенерировать скрипт загрузки с помощью команды `mix firmware.gen.script`.
+Эта команда сгенерирует новый скрипт `upload.sh`, который мы сможем запустить для обновления прошивки.
+
+Если после предыдущего шага сеть функционирует, все готово.
+
+Чтобы обновить настройки, проще всего использовать `mix firmware && ./upload.sh 192.168.88.2`: первая команда создает обновленную прошивку, а вторая отправляет ее по сети и перезагружает устройство.
+Наконец-то вам больше не придется постоянно вставлять и вынимать SD-карты из устройства!
+
+_Подсказка: `ssh 192.168.88.2` предоставляет вам оболочку IEx на устройстве в контексте приложения._
+
+_Устранение неполадок: Если в вашей домашней папке нет существующего ключа SSH, вы получите ошибку `No SSH public keys found in ~/.ssh.`.
+В этом случае вам нужно будет запустить `ssh-keygen` и повторно записать прошивку, чтобы использовать функцию обновления по сети._
+
+## Настройка управления светодиодами
+
+Для взаимодействия со светодиодами вам необходимо установить пакет [nerves_leds](https://github.com/nerves-project/nerves_leds), что делается путем добавления зависимости `{:nerves_leds, "~> 0.8", targets: @all_targets},` в файл `mix.exs`.
+
+После установки зависимости вам необходимо настроить список светодиодов для данного устройства.
+Например, для всех моделей Raspberry Pi на плате есть только один светодиод: `led0`.
+Давайте используем его, добавив строку `config :nerves_leds, names: [green: "led0"]` в `config/config.exs`.
+
+Для других устройств вы можете взглянуть на [соответствующую часть проекта nerves_examples](https://github.com/nerves-project/nerves_examples/tree/main/hello_leds/config).
+
```

Comment:
Link to `hello_leds` example is deprecated due to this [PR](https://github.com/nerves-project/nerves_examples/pull/251).
Should we use [this link](https://github.com/nerves-project/nerves_examples/tree/main/blinky/config) instead in all locales?

Reply:
Howdy @dmitry-sidorov! Thanks for catching this, I think that'd be a great idea. Would you be willing to make that change to the other lessons?

Reply:
Hey, man! Sure, I am on it! Should I add these changes here or create another PR?

Reply:
Whichever you prefer @dmitry-sidorov 😀 Really appreciate the help 💪

Reply:
@doomspork Check out [PR](https://github.com/elixirschool/elixirschool/pull/3025)

Reply:
You're awesome, thank you @dmitry-sidorov!